how-to-run-a-local-server-with-firebase-serve/readme.md

```diff
@@ -0,0 +1,37 @@
+> **Goal of this guide:** Show you how you can run a local server using firebase serve
+
+# How to run a local server using firebase serve
+
+When creating websites you need to see what your new implementation looks like before displaying it online. No good having your visitors see all your trial and errors, is it ;)
+
+One way to avoid this is by 'running a server' locally. You can change whatever you like and you'll be the only person who'll see it––except for maybe your colleague who'll be amazed by your inclusion of a purple horse on your website!
+
+## If you've run a local server before
+
+> Skip this bit if this is your first time running a local server
+
+1. [Navigate in the terminal](http://www.macworld.com/article/2042378/master-the-command-line-navigating-files-and-folders.html) to the folder of the website you want to change.
+2. Navigate to the folder web ––```cd web```
```

Comment:
This is project specific so this should be in the project's `readme.md`, together with whether you use `bower`